opendatahub-io/opendatahub-operator
# Revert "kserve: add grpconly envoy filter (#888)"

This reverts commit 48e0b8bad9bbae951731a36c6e17bf26981fed7b (#888).

This was a temporary fix for https://issues.redhat.com/browse/RHOAIENG-165 which does not affect caikit+tgis, tgis or ovms anymore.


testing instructions:
1. check out https://github.com/opendatahub-io/opendatahub-operator/pull/888 and spin it up
2. check that the kserve-temporary-fixes feature has been added and that grpc-only EnvoyFilter exists.
3. Spin this PR up
4. check that the kserve-temporary-fixes feature has been deleted and that the grpc-only EnvoyFilter has been deleted
